scripts/demo-content.py

```diff
@@ -26,6 +26,13 @@ def main():
         users_list.append(user)
 
     for user in users_list:
+        for x in range(3):
```

Comment:
in questo modo stai creando 3 indirizzi per ogni utente. Creiamone in numero random da 1 a 3 indirizzi